client/src/containers/string-avatar/type.d.ts

```diff
@@ -0,0 +1,4 @@
+export type stringAvatarProps = {
+  className?: string;
```

Comment:
```suggestion
export type stringAvatarProps = React.HTMLAttributes<HTMLElement> & {
```
you don't need to add types for className prop, it's better to extend from defaults